### PR TITLE
Feature/516-522-521

### PR DESCRIPTION
**Pull request by jenbutongit**
**PR includes:**

* changes for [522-learning-record-update-the-font](https://trello.com/c/MZ4Jo48l/522-learning-record-update-the-font) - added fonts and fixed sizes _..mostly..probably..hopefully_
* changes for [516-update-suggestions-for-you-tiles](https://trello.com/c/AyqgpyiK/516-update-suggestions-for-you-tiles)
* changes for [521-suggestions-for-you-page-update-design-to-reflect-multiple-professions](https://trello.com/c/9h97uKxQ/521-suggestions-for-you-page-update-design-to-reflect-multiple-professions)

**big stuff to note**
*I've added a function to the User class which should translate the stringified array from the professions field because wso2 wont accept array :( to an array so we don't have to constantly check .includes and do .split()
* I've also created a class which constructs the query string for the catalogue api, and then you can .serialize() on it. Not entirely how typescript struct/classes should work.. hopefully it makes sense and is .. _typescripty_
* some logic for the suggestions stuff. On home: 5 courses suggested by their `area of work` only. Then 1 from their department. On the suggest page, it is one section (max 6 courses) per area of work, then one section for their department.

for @williampri 
* bug fixes for the profile page :)

### CODE REVIEW

File: src/ui/component/Tile.html [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+
+	        {{i18n('areas-of-work')[aow]}}{{course.getAreasOfWork().length !== index + 1 ? ', ' : ''}}
+	    {{/each}}
+	{{/if}}
+	</p>
 	<ul class="tile__properties">
-		<li class="tile__property">{{course.getType()}}</li>
+            <li class="tile__property">
+                <span class="tile__key">Type</span>
+                <span class="tile__value">{{course.getType()}}</span>
```
Comment (superafroman) [inline src/ui/component/Tile.html]: Should be passed to i18n

File: src/ui/component/Tile.html [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+            <li class="tile__property">
+                <span class="tile__key">Duration</span>
+                <span class="tile__value">{{course.getDuration()}}</span>
+            </li>
+
 
-		<li class="tile__property">{{course.getDuration()}}</li>
+            <li class="tile__property">
+                <span class="tile__key">Cost</span>
+                <span class="tile__value">&pound;{{course.price ? course.price : 'Free'}}</span>
```
Comment (superafroman) [inline src/ui/component/Tile.html]: 'Free' should be passed to i18n

File: src/ui/component/Tile.html [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
 <div class="tile">
 	<h4 class="heading-small">
 		<a href="/courses/{{course.id}}">
 			{{course.title}}
 		</a>
 	</h4>
 
-	<p class="tile__attribution">Recommended by profession</p>
+	<p class="tile__attribution">Work area:
+	{{#if course.getAreasOfWork().length}}
```
Comment (superafroman) [inline src/ui/component/Tile.html]: No need to check length as effectively done by loop.

File: src/ui/component/Tile.html (hunk 2 above)
Comment (superafroman) [reply src/ui/component/Tile.html]: Really, everything should be - so 'Cost', 'Duration', etc. 😄

**Comment by williampri** [comment]
Any update on this PR?

**Comment by jenbutongit** [comment]
[Update the font/typography](https://trello.com/c/MZ4Jo48l/522-update-the-font-typography)

**Comment by jenbutongit** [comment]
[Update 'suggestions for you' tiles](https://trello.com/c/AyqgpyiK/516-update-suggestions-for-you-tiles)

**Comment by jenbutongit** [comment]
[Update 'suggestions for you' tiles](https://trello.com/c/AyqgpyiK/516-update-suggestions-for-you-tiles)

**Comment by jenbutongit** [comment]
[Suggestions for you page: update design to reflect multiple professions](https://trello.com/c/9h97uKxQ/521-suggestions-for-you-page-update-design-to-reflect-multiple-professions)

**Pull request merged**